ext.gradle

```diff
@@ -18,12 +18,12 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-final def JDBC_VERSION = "0.9.78-SNAPSHOT"
+final def JDBC_VERSION = "0.10.0"
 
 ext {
     jdbcVersion = JDBC_VERSION
     coreVersion = "0.10.5-SNAPSHOT"
```

Comment:
Let's modify this file and use single quote instead of `"` for versions.

Reply:
Done.